checker/run

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -uex
+
+. "$(dirname "$0")"/../options
+
+if [ -z "$pattern_extractor" ] | [ -z "$message" ]; then
+    echo "Required fields pattern_extractor and message not found in ../options"
+    exit 1
+fi
+
+new_value=$(eval "$pattern_extractor" | tr -d '\r')
+if [ -z "$new_value" ]; then
+    echo "$pattern_extractor didn't found anything"
+    exit 1
+fi
+
+old_value=$(git log -1 --pretty=%B | head -1 | perl -pe "s|$message \((.*)\)|\1|")
```

Comment:
I'm used to it's regexp syntax but I agree it's overkill here, switched to sed (the grouping in the regexp is still readable in this simple case).